jetty-core/jetty-client/src/main/java/org/eclipse/jetty/client/transport/HttpSender.java

```diff
@@ -328,10 +329,15 @@ public void proceed(HttpExchange exchange, Throwable failure)
         if (failure == null)
             contentSender.iterate();
         else
-            anyToFailure(failure);
+            anyToFailure(failure, true);
     }
 
     public void abort(HttpExchange exchange, Throwable failure, Promise<Boolean> promise)
+    {
+        externalAbort(failure, promise);
+    }
+
+    private boolean localAbort(Throwable failure)
```

Comment:
aborts can be local, internal or external?   Can you explain that in a comment/javadoc somewhere?

would this method be better named `doAbort` ?